src/main/java/org/sagebionetworks/bridge/models/schedules/ActivityScheduler.java

```diff
@@ -51,14 +51,14 @@
             // Start time is event time, with added delay if it's present.
             DateTime startTime = oneEventTime;
             if (schedule.getDelay() != null) {
-                startTime = oneEventTime.plus(schedule.getDelay());
+                startTime = oneEventTime.withZone(context.getRequestTimeZone()).plus(schedule.getDelay());
             }
```

Comment:
If delay is null (which can be fairly common), then we don't set the time zone. It might be cleaner to put the withZone() on line 65 (scheduleWindowList.add(...)), which should guarantee that we never add something without ensuring its timezone.

Reply:
I've removed the .withZone() calls here because this is already being set when the event map of event times is being constructed, and to your point in the comment below, we should verify there that this is being set correctly.